duffel_api/http_client.py

```diff
@@ -12,7 +14,7 @@ class ClientError(Exception):
 class ApiError(Exception):
```

Comment:
💭 We should probably add unit tests for this class and others in this file at some future point.